### DISCUSSION TITLE
Designer and ForEach

### DISCUSSION THREAD

**Discussion by tomy2105** [elsa-workflows/elsa-core#796] [Q&A]
I'm probably missing something but don't understand how ForEach should work in designer.
ForEach activiity gives me empty drop down list, I cannot add variable in here or anything :(.
If someone could please explain :).

Same goes with Switch activity. It has empty dropdown for "Cases" and I don't understand how to use it.

**Comment by craigfowler** [comment]
@sfmskywalker - I think this one needs your brain 😁

**Comment by sfmskywalker** [comment]
You’re not missing anything. You are looking at a work on progress. The designer has many quirks and missing pieces at the moment.
